vmdb/app/models/host_service_group.rb

```diff
@@ -0,0 +1,11 @@
+class HostServiceGroup < ActiveRecord::Base
+  has_many :filesystems, :dependent => :nullify
```

Comment:
@Fryguy it models that HostServiceGroup has configuration, which consists of many configuration files.

That is e.g. /etc/nova/nova.conf for all nova services, there can also be configuration file for specific service, but we will not model it now